web/src/views/DandiMetaViewer.vue

```diff
@@ -29,7 +29,27 @@
                   This dataset has not been published!
                 </v-chip>
               </v-card-title>
-              <v-card-actions>
+              <v-list dense v-if="meta.identifier" class="py-0">
+                <v-list-item>
+                  <v-list-item-content>
+                    Identifier: {{ meta.identifier }}
+                  </v-list-item-content>
+                  <!-- <v-list-item-action class="ma-0 mr-1">
+                    <v-btn icon>
+                      <v-icon>mdi-content-copy</v-icon>
+                    </v-btn>
+                  </v-list-item-action>
+                  <v-spacer /> -->
+                </v-list-item>
+                <v-list-item>
+                  <v-list-item-content>
+                    <a :href="permalink">
+                      {{ permalink }}
+                    </a>
+                  </v-list-item-content>
+                </v-list-item>
+              </v-list>
+              <v-card-actions class="py-0">
                 <v-btn
                   icon
                   @click="$router.push(`/collection/${selected.parentId}`)"
```

Comment:
I realize this isn't part of this PR but since we are already doing this, let's go ahead and replace `@click` with `:to` here.